src/scoreboard/controllers/controller.js

```diff
@@ -0,0 +1,105 @@
+import Mustache from 'mustache';
```

Comment:
This controller is completely copied over from the 'normal' game. Is this fine or should I extend the one in the 'normal' game.

If we choose to extend we need to keep the GAME variable.

Reply:
Re-use what you need, but drop everything else. That should be relatively easy to do: just copy the parts you need.

Reply:
We need everything, so I will just leave it in there.